.circleci/config.yml

```diff
@@ -121,7 +126,6 @@ commands:
       study_key:
         type: string
     steps:
-      - set-deployment-environment
```

Comment:
Looks like when circle ci executed `deploy-stored-build`, `set-deployment-environment` was called twice: in the `deploy-stored-build` and then in the `deploy-exploded-build`.